<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="score_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>620</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Programme Bold</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="rejouerButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>330</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Programme Bold</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string> Rejouer</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../assets/rejouer.png</normaloff>../assets/rejouer.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="percentage_label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>481</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Programme Regular</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
